src/features/game/expansion/components/Water.tsx

```diff
@@ -65,7 +65,7 @@ export const Water: React.FC<Props> = ({ level }) => {
           />
         </MapPlacement>
 
-        <MapPlacement x={-2} y={offset + 2}>
+        <MapPlacement x={-2} y={offset + 2} width={64 * PIXEL_SCALE}>
```

Comment:
The width on the `MapPlacement` refers to how many squares wide it will be.

This can be `1` to ensure it is a square wide.

<img width="937" alt="Screen Shot 2022-10-12 at 1 06 46 pm" src="https://user-images.githubusercontent.com/11745561/195232352-0da62703-afb5-4b6e-a7c7-40f3ac7c4617.png">